README.md

```diff
@@ -10,4 +10,5 @@ See:
   * [php](php) example
   * [python](python) example
   * [rust](rust) example
+  * [sqlalchemy](sqlalchemy) example
```

Comment:
I'd not spotted this was NOT under the Python directory. Sorry for not catching this earlier.

Please move under the Python dir, so we get `python/sqlalchemy`, then move this bullet as a sub-bullet (indented).

Reply:
Done